test/unit/gateways/orbital_test.rb

```diff
@@ -567,7 +582,7 @@ def test_american_requests_adhere_to_xml_schema
     response = stub_comms do
       @gateway.purchase(50, credit_card, :order_id => 1, :billing_address => address)
     end.check_request do |endpoint, data, headers|
-      schema_file = File.read("#{File.dirname(__FILE__)}/../../schema/orbital/Request_PTI54.xsd")
+      schema_file = File.read("#{File.dirname(__FILE__)}/../../schema/orbital/Request_PTI77.xsd")
```

Comment:
It felt appropriate to update the XSD used for tests to be consistent with the API version update. I left this as a distinct commit in case we decide against doing so.